programmers/두-개-뽑아서-더하기.js

```diff
@@ -0,0 +1,16 @@
+function solution(numbers) {
+    
+    const len = numbers.length;
+    let result = [];
+    let add;
```

Comment:
```suggestion
    let addNum = 0;
```
변수를 선언할 때 초기값을 줘야합니다. 초기값을 줘야 이 변수가 어떤 형으로 쓰일 건지 알 수 있기 때문이에요
그리고 add 보다는 addNum으로 해서 더한 숫자라고 알려주는게 좋겠죠